src/index.ts

```diff
@@ -1,28 +1,37 @@
-import {APIClient, Asset, AssetType, Name, NameType} from '@wharfkit/antelope'
-import * as SystemTokenContract from './contracts/system.token'
+import {Action, APIClient, Asset, AssetType, Name, NameType} from '@wharfkit/antelope'
 import {Contract, ContractKit} from '@wharfkit/contract'
 
 interface TokenOptions {
     client: APIClient
-    tokenSymbol?: Asset.SymbolType
     contract?: Contract
 }
 
 export class Token {
-    readonly contract: Contract
-    readonly kit: ContractKit
+    client: APIClient
+    contractKit: ContractKit
+    contract?: Contract
 
-    constructor({contract, client}: TokenOptions) {
-        this.kit = new ContractKit({
-            client,
+    constructor({client, contract}: TokenOptions) {
+        this.client = client
+        this.contract = contract
+        this.contractKit = new ContractKit({
+            client: client as any,
         })
-        this.contract = contract || new SystemTokenContract.Contract({client})
     }
 
-    async transfer(from: NameType, to: NameType, amount: AssetType, memo = '') {
+    async getContract(contractName?: string) {
+        if (contractName) {
+            return this.contractKit.load(contractName)
+        }
+        return this.contract || this.contractKit.load('eosio.token')
```

Comment:
After upgrading to the latest contract kit, I was seeing a bunch of type errors. I figured that it wasn't needed since we now need to support loading the contract. In hindsight, I'm not sure it that made sense though so I'm fine putting it back.

Reply:
Added the latest version of the system contract 👍